pkg/broker/file.go

```diff
@@ -70,14 +71,7 @@ func WriteInfoToFile(restConfig *rest.Config, brokerNamespace, ipsecFile string,
 		data.CustomDomains = &customDomains
 	}
 
-	err = data.writeToFile(InfoFileName)
-	if err != nil {
-		return status.Error(data.writeToFile(InfoFileName), "error saving broker info")
```

Comment:
This also avoids a double call to `data.writeToFile()` on errors.

Reply:
Thanks for the cleanup. I don't know what I was thinking 😄